syft/pkg/cataloger/rust/package.go

```diff
@@ -46,6 +47,8 @@ func newPackageFromAudit(dep *rustaudit.Package, locations ...file.Location) pkg
 		Language:  pkg.Rust,
 		Type:      pkg.RustPkg,
 		Locations: file.NewLocationSet(locations...),
+		// no attempt is made by the parser function to resolve dependencies
```

Comment:
I think the attempt is made now that #3500 is merged.

Reply:
I'll handle this on the rebase